CHANGELOG.md

```diff
@@ -0,0 +1,140 @@
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-149'>AUTH-149</a>] -         Identify All Artifacts that need to be localized
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-150'>AUTH-150</a>] -         Document instructions for configuring LDAP sync
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-151'>AUTH-151</a>] -         Test LDAP sync
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-153'>AUTH-153</a>] -         Document instructions for configuring SAML
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-154'>AUTH-154</a>] -         Compliance with Alfresco Security Policy
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-155'>AUTH-155</a>] -         Complete and Update Documentation for Deploying the Identity Service
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-158'>AUTH-158</a>] -         Publish Artifacts
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-159'>AUTH-159</a>] -         Naming
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-160'>AUTH-160</a>] -         Release Notes
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-162'>AUTH-162</a>] -         Automate regression testing for LDAP Authentication
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-184'>AUTH-184</a>] -         Create a list of 3rd Party Software Dependencies
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-193'>AUTH-193</a>] -         Stable Infrastructure Chart (with Identity Service)
+</li>
+</ul>
+
+<h2>Bug
+</h2>
+<ul>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-120'>AUTH-120</a>] -         Pass the secret name as values to identity service 0.2 helm chart
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-144'>AUTH-144</a>] -         alfresco-anaxes-hello-world documentation steps inconsistency
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-145'>AUTH-145</a>] -         alfresco-anaxes-hello-world-ui throws SyntaxError in browser console
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-152'>AUTH-152</a>] -         ACS6: New identity-service authentication sub-system can freeze server
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-171'>AUTH-171</a>] -         Token request with SAML provider results with &quot;upstream sent too big header&quot;
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-172'>AUTH-172</a>] -         Identity Service Ingress is Broken Due to Truncation Differences
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-178'>AUTH-178</a>] -         Empty Realm in Identity Service
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-181'>AUTH-181</a>] -         Enable per-namespace host names in Bamboo-driven SAML tests
```

Comment:
This is part of our CI pipeline, not visible to customers. I suggest that we remove this.

Reply:
I removed AUTH-181 from the list